chef/cookbooks/provisioner/templates/default/autoyast.xml.erb

```diff
@@ -268,7 +268,7 @@ sync
       <package>biosdevname</package>
       <package>netcat</package>
       <package>ruby2.1-rubygem-chef</package>
-<% if @is_ses -%>
+<% if @is_ses and @target_platform_version == "12.0" -%>
```

Comment:
I'm not sure if this breaks something else this was my first idea to fix the issue. Maybe @tserong has a better one.